packages/concerto-core/types/lib/typenotfoundexception.d.ts

```diff
@@ -7,14 +7,6 @@ export = TypeNotFoundException;
  * @memberof module:concerto-core
  */
 declare class TypeNotFoundException extends BaseException {
-    /**
-     * Constructor. If the optional 'message' argument is not supplied, it will be set to a default value that
-     * includes the type name.
-     * @param {string} typeName - fully qualified type name.
-     * @param {string|undefined} message - error message.
-     * @param {string} component - the optional component which throws this error
-     */
-    constructor(typeName: string, message: string | undefined, component: string);
```

Comment:
On the main I still see constructor on `TypeNotFoundException.js` but why is this getting deleted here? Is this auto generated?

Reply:
Yes this is actually auto generated after I run build command